File: tests/corpus/positive-fixtures/QA-SE-001/SleepBeforeLookup.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;

public class SleepBeforeLookup {
    public void openReports(WebDriver driver) {
        driver.get("https://reports.example.test/daily");
        try { Thread.sleep(3000); } catch (InterruptedException ignored) {}
        WebElement table = driver.findElement(By.id("daily-table"));
        assert table.isDisplayed();
    }
}

File: tests/corpus/positive-fixtures/QA-SE-001/SleepBeforeLookup.py
import time

from selenium import webdriver
from selenium.webdriver.common.by import By


def test_daily_report(driver):
    driver.get("https://reports.example.test/daily")
    time.sleep(3)
    table = driver.find_element(By.ID, "daily-table")
    assert table.is_displayed()
